codechecker_lib/analyzers/analyzer_base.py

```diff
@@ -118,3 +111,22 @@ def get_analyzer_checkers(self, config_handler, env):
         return the checkers available in the analyzer
         """
         pass
+
+    @staticmethod
+    def run_proc(command, env=None, cwd=None):
+        """
+        Just run the given command and return the returncode
+        and the stdout and stderr outputs of the process.
+        """
+
+        cmd = shlex.split(command, posix=False)
```

Comment:
After some offline discussion: posix=False is required to avoid removing some double quotes from the command. I think this worth a comment. Otherwise LGTM.